modules/ROOT/pages/datagraph-qsg.adoc

```diff
@@ -99,7 +99,7 @@ The default query method for an object type is the method that always returns a
 +
 image::datagraph-qsg-primary-key.png[Primary key set to identifier string]
 +
-The primary key is one field of your object type that uniquely identifies a single record of that object type.
+The primary key is one field of your object type that uniquely identifies a single record of that object type. DataGraph also supports object types that use _composite keys_, but in ths guide you'll just one primary key.
```

Comment:
We don't typically refer to "this guide" or address our audience as learners, per se. I'd remove that last bit.

Reply:
If you keep it, it needs repair:
```suggestion
The primary key is one field of your object type that uniquely identifies a single record of that object type. DataGraph also supports object types that use _composite keys_, but in this guide, you'll use just one primary key.
```